main/guides/getting-started/README.md

```diff
@@ -297,4 +297,17 @@ When the transaction is complete you will notice some IST has been debited from
 
 ![Finished transaction](./assets/new_006.png)
 
-Congratulations! You got your first Agoric dapp up and running!
+Congratulations! You got your first Agoric dapp up and running! Now that you've got your first Agoric dapp up and running, let's focus on some key takeaways from this exercise:
+- Starting an Agoric Chain: You have learned the process to initialize and start a local Agoric blockchain. 
+```sh
+yarn start:docker
+```
```

Comment:
hm. I wouldn't call that **the** process to start a local chain. `yarn start:docker` is not a general purpose mechanism at all, and we didn't really teach them how it works.

Maybe...

----

Congratulations! You got your first Agoric dapp up and running!

Key takeaways:

 - start a local Agoric blockchain
 - deploy a smart contract
 - make an offer, using a web ui and Keplr extension
----

Hm... I don't love that either.

Reply:
I think we need @sufyaankhan's input on this one. The code examples I added in this section were the lines from this doc - that's how I understood @sufyaankhan's request in Item 5 of #919